### ISSUE TITLE
Updating Pockyt for 3.8

### ISSUE THREAD

**Issue by m040601** [arvindch/pockyt#14]
similar issue to last year python 3.7 update, https://github.com/arvindch/pockyt/issues/13

This is just to check if pockyt, itself,  is compatible with python 3.8, and it doesnt need python 2 at all.
It was working just fine until November 2019. 
With ArchLinux update to python 3.8 this AUR package, https://aur.archlinux.org/packages/pockyt/

now breaks with:
```

This version of Python [3.8.0] is unsupported ! 
Please use Python 2.7.x, 3.4.x, 3.5.x, or 3.6.x !


```

Thanks for your work on this usefull tool. Keep using it after all these years !

**Comment by wedens** [comment]
Meanwhile you can apply the [patch](https://patch-diff.githubusercontent.com/raw/arvindch/pockyt/pull/15.patch) from https://github.com/arvindch/pockyt/pull/15.


PKGBUILD changes (assuming you've saved the patch to `python_3_8.patch` in PKGBUILD directory):
```
diff --git a/PKGBUILD b/PKGBUILD
index a3aa895..d576721 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -7,12 +7,19 @@ depends=('python' 'python-parse')
 makedepends=('python' )
 license=('GPLv3+')
 arch=('any')
-source=("https://github.com/arvindch/pockyt/archive/$pkgver.tar.gz")
-md5sums=('295ba973b0bdcdb5578bb38d866cf86d')
+source=(
+  "https://github.com/arvindch/pockyt/archive/$pkgver.tar.gz"
+  "python_3_8.patch"
+)
+md5sums=(
+  '295ba973b0bdcdb5578bb38d866cf86d'
+  'SKIP'
+)
 
 
 build() {
     cd $srcdir/pockyt-$pkgver
+    patch --strip=1 < "${srcdir}/python_3_8.patch"
     python setup.py build
 }
```

**Comment by arvindch** [comment]
Fixed in https://github.com/arvindch/pockyt/pull/15.

I will push a new PyPi release on the weekend. Cheers!

**Comment by arvindch** [comment]
Also, thanks for the support 😄 ! Always happy to hear that people find this useful, years later.

**Comment by schtibe** [comment]
@arvindch Thanks for this package! I don't want to pressure you, just wondering: did you manage to push a new release?

**Comment by arvindch** [comment]
Note: With https://github.com/arvindch/pockyt/commit/cbe87d6feeede8922f122a24ce656d74d176b476, all new versions of Python should be supported. Pockyt doesn't do anything wonky or fancy, so I assume this assumption should work for the forseeable future.

@schtibe - I have released pockyt 1.3.1 today with all the latest changes. - https://pypi.org/project/pockyt/1.3.1/

**Comment by schtibe** [comment]
Thank you @arvindch
